torchgeo/models/resnet.py

```diff
@@ -16,7 +16,51 @@
 __all__ = ["ResNet50_Weights", "ResNet18_Weights"]
 
 _zhu_xlab_transforms = AugmentationSequential(
```

Comment:
If you want to add links to where the transforms come from, you can use:

https://github.com/zhu-xlab/SSL4EO-S12/blob/d2868adfada65e40910bfcedfc49bc3b20df2248/src/benchmark/transfer_classification/linear_BE_moco.py#L167

(I think they are all the same?)

Normalization seems to be either divide by 10K or apply channel-wise normalizations: 

https://github.com/zhu-xlab/SSL4EO-S12/blob/d2868adfada65e40910bfcedfc49bc3b20df2248/src/benchmark/transfer_classification/datasets/EuroSat/eurosat_dataset.py#L95,L97

We could try both and default to whichever works better.

Reply:
I think @calebrob6 ran an experiment where he found that divide by 10K seems to work well. Not sure if he has tried anything else. 

I also understand that the transforms are the same across their models.

Reply:
Let's also add a comment for https://github.com/zhu-xlab/SSL4EO-S12/blob/d2868adfada65e40910bfcedfc49bc3b20df2248/src/benchmark/transfer_classification/datasets/EuroSat/eurosat_dataset.py#L97